apps/test/unit/containedLevelsTest.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
what's the reason for the file name change?

Reply:
Just trying to leave the world a better place. Other files with tests tend to end in Test.js, and I noticed you didn't find the test file right away because you were looking for containedLevelsTest.js.